cmd/postpolicyform.go

```diff
@@ -265,6 +265,22 @@ func checkPostPolicy(formValues http.Header, postPolicyForm PostPolicyForm) erro
 	if !postPolicyForm.Expiration.After(UTCNow()) {
 		return fmt.Errorf("Invalid according to Policy: Policy expired")
 	}
+	// check all formValues appear in postPolicyForm or return error. #https://github.com/minio/minio/issues/17391
+	checkHeader := map[string][]string{}
+	ignoreKeys := map[string]bool{}
```

Comment:
Can we just use a slice of string to maintain list of keys to ignore and check keys against this list?

Reply:
it can set by `X-Amz-Ignore-Foo` @shtripat